src/NuGet.Services.EndToEnd/Support/TestSettings.cs

```diff
@@ -28,86 +31,148 @@ public enum Mode
         /// </summary>
         public static bool DefaultAggressivePush => true;
 
+        /// <summary>
+        /// A list of default test settings per environment.
+        /// </summary>
+        public static readonly Dictionary<Mode, TestSettings> DefaultTestSettings = new Dictionary<Mode, TestSettings>
+        {
+            {   Mode.Dev,
+                new TestSettings(
+                             DefaultAggressivePush,
+                             "https://dev.nugettest.org",
+                             "https://api.dev.nugettest.org/v3-index/index.json",
+                             new List<string>(),
+                             "fa133685-cfc3-43f3-81c7-0a36d0969987",
```

Comment:
There's also a unit test that asserts that this API key is not test. Are those passing?